eq-author/src/App/page/Design/answers/BasicAnswer/index.js

```diff
@@ -34,9 +60,21 @@ export const StatelessBasicAnswer = ({
   getValidationError,
   type,
   optionErrorMsg,
+  multipleAnswers,
 }) => {
+  let [toggled, setToggled] = useState(false);
   const errorMsg = buildLabelError(MISSING_LABEL, `${lowerCase(type)}`, 8, 7);
 
+  useEffect(() => {
+    if (multipleAnswers) {
+      setToggled(false);
+    }
+  }, [multipleAnswers]);
```

Comment:
What's the rationale behind this `useEffect`? Just curious

Reply:
mmm - interesting question - initially the toggle refused to turn off without it, but now seems to work fine without it

Reply:
ahhh - thats it - now I remember -  if the toggle is ON and then you 'add another answer' - this is to re-render the toggle with updated props and hence turn off...